### DISCUSSION TITLE
ArnoldProceduralCustom can be translated only as Instanceable

### DISCUSSION THREAD

**Discussion by Ark-Dem** [Autodesk/maya-usd#3538] [Q&A]
I'm exporting a mesh with static fur generated by Yeti plugin. It gets exported as ArnoldProceduralCustom and mostly works, but has two issues:

(1) "USD Export" plugin doesn't export path to cache file and "samples" attribute, I have to add these lines manually for fur to work:
string arnold:filename = "C:/cache.fur"
float[] arnold:samples = [1]

"Arnold-USD" plugin adds them properly and works right away (though it has some other issues that makes adding these lines manually a lesser evil for me, so, I keep using "USD Export").

And no chance for relative path here at all (but I suppose that's Yeti problem).

It would be great to make "USD Export" respect and export these attributes.

(2) When I translate Xform prims above the fur it doesn't move... unless I load the asset as a payload and make it instanceable... but then I lose the ability to move the elements inside it separately.

Is that possible to make ArnoldProceduralCustom understand translations without the hassle of payloading and instanceabling it?

**Comment by Ark-Dem** [comment]
After more tests I've figured out that it's actually yeti cache that holds fur in place, not prim's instanceable state... on the opposite, instanceable state makes it move where it shouldn't... anyway, question (2) is irrelevant now.

**Comment by wallworm** [comment]
Thanks for sharing. We will look into this.
